src/main/java/org/opensearch/flowframework/model/Template.java

```diff
@@ -32,19 +32,19 @@
 public class Template implements ToXContentObject {
 
     /** The template field name for template name */
-    public static final String NAME_FIELD = "name";
+    private static final String NAME_FIELD = "name";
```

Comment:
Why are these all being changed to `private`?
 - we should be able to use them in testing
 - we should be able to use them in Workflow Steps to create them

Reply:
For workflow step we are fetching it from [here](https://github.com/opensearch-project/opensearch-ai-flow-framework/blob/main/src/main/java/org/opensearch/flowframework/common/CommonValue.java). These are specific to Template.

Reply:
> For workflow step we are fetching it from [here](https://github.com/opensearch-project/opensearch-ai-flow-framework/blob/main/src/main/java/org/opensearch/flowframework/common/CommonValue.java).

Then delete it here and fetch it from there. :)